api/models.py

```diff
@@ -1,38 +1,41 @@
-from django.db import models
 from django.contrib.auth.models import User
+from django.db import models
+
 
-# Create your models here.
 class Video(models.Model):
     title = models.CharField(max_length=100)
     created_at = models.DateTimeField(auto_now_add=True)
     viewed_at = models.DateTimeField(null=True)
 
-#All possible relationships
-#(true,true), (true, null), (false, true/null/false), (null,null)
-#Note that we don't need to store (null,null)
+
+# All possible relationships
+# (true,true), (true, null), (false, true/null/false), (null,null)
+# Note that we don't need to store (null,null)
 class MatchStatus(models.Model):
     primary_user = models.ForeignKey(User, on_delete=models.CASCADE, related_name="+")
```

Comment:
So the original version of this was actually incorrect. Since we decided to store entries when the initial query is made, there WILL be (null,null). So primary_user needs a null=True